Resources/views/line/video.html.twig

```diff
@@ -0,0 +1,27 @@
+{% set videoType = content.getField( 'file' ).value.mimeType|default( 'video/mp4' ) %}
+
```

Comment:
yes, but my point is if you're in a folder you are not adding that. you only list the videos but never set that setting you set in full view, right? 
in legacy we had something like ```run-once``` but dunno if we have something similar here.